omdbscore.py

```diff
@@ -1,14 +1,53 @@
 from privatekeys import omdb_key
 import requests
 import urllib
+import json
 
-baseUrl = 'http://www.omdbapi.com/?apikey=[yourkey]&'
-query = 'Batman v superman'
-params = {
-    'apiKey' : omdb_key,
-    's' : query
-}
+def findOmdbScoreFromId(imdbID):
+    baseUrl = 'http://www.omdbapi.com/'
+    params = {
+        'apiKey' : omdb_key,
+        'i' : imdbID
+    }
 
-url = baseUrl + '?' + urllib.parse.urlencode(params)
-movieInfo = requests.get(url).content
-print(movieInfo)
\ No newline at end of file
+    url = baseUrl + '?' + urllib.urlencode(params)
```

Comment:
Should be urllib.parse.urlencode(params) for python3!

Reply:
Oh oops, i was using 2.7